webapp/src/dialogs.tsx

```diff
@@ -68,7 +74,7 @@ export function showAboutDialogAsync(projectView: pxt.editor.IProjectView) {
 }
 
 
-function renderCompileLink(cs: pxt.TargetCompileService) {
+function renderCompileLink(variantName: string, cs: pxt.TargetCompileService) {
```

Comment:
Is `variantName` intended to be used anywhere in here? Seems fine as it's taking the info from the cs

Reply:
right, i taught i would never but eventually i didn't. The repo description is actually enough.